community/consistency-check/src/test/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporterTest.java

```diff
@@ -237,7 +236,7 @@ private void assertConsistent( File storeDir ) throws ConsistencyCheckIncomplete
 
     protected String getFormatName()
     {
-        return StandardV3_0.NAME;
+        return StandardV3_0_7.NAME;
```

Comment:
(This comment applies to all similar changes): should we have a notion of "latest" standard format to not have to make wide-spread changes like this and potentially forget some?